### PR TITLE
🔀 :: (#3)  Pass 인증 기능 구현

### LINKED ISSUE #3: ✨ :: Pass 인증 기능
Pass 인증 기능을 추가합니다.
## 참고사항
Rolls-Royce auth 부분을 참고합니다.

### PR DESCRIPTION
**Pull request by HyunSu1768**
close #3

### CODE REVIEW

File: src/main/kotlin/hs/kr/equus/user/domain/auth/entity/PassInfo.kt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package hs.kr.equus.user.domain.auth.entity
+
+import org.springframework.data.annotation.Id
+import org.springframework.data.redis.core.RedisHash
+import org.springframework.data.redis.core.TimeToLive
+import org.springframework.data.redis.core.index.Indexed
+
+@RedisHash
+data class PassInfo(
+
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/entity/PassInfo.kt:10]: 🚫 **[ktlint]** <standard:no-blank-line-in-list> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected blank line(s) in value parameter list

File: src/main/kotlin/hs/kr/equus/user/domain/auth/entity/PassInfo.kt [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    @Id
+    val name: String,
+
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/entity/PassInfo.kt:13]: 🚫 **[ktlint]** <standard:no-blank-line-in-list> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected blank line(s) in value parameter list

File: src/main/kotlin/hs/kr/equus/user/domain/auth/entity/PassInfo.kt [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    @Indexed
+    val phoneNumber: String,
+
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/entity/PassInfo.kt:16]: 🚫 **[ktlint]** <standard:no-blank-line-in-list> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected blank line(s) in value parameter list

File: src/main/kotlin/hs/kr/equus/user/domain/auth/entity/PassInfo.kt [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    @TimeToLive
+    val ttl: Long
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/entity/PassInfo.kt:18]: 🚫 **[ktlint]** <standard:trailing-comma-on-declaration-site> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing trailing comma before ")"

File: src/main/kotlin/hs/kr/equus/user/domain/auth/entity/repository/PassInfoRepository.kt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+package hs.kr.equus.user.domain.auth.entity.repository
+
+import hs.kr.equus.user.domain.auth.entity.PassInfo
+import org.springframework.data.repository.CrudRepository
+
+interface PassInfoRepository : CrudRepository<PassInfo, String> {
+
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/entity/repository/PassInfoRepository.kt:7]: 🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line

File: src/main/kotlin/hs/kr/equus/user/domain/auth/exception/InvalidOkCertConnectException.kt [hunk 6]
```diff
@@ -0,0 +1,8 @@
+package hs.kr.equus.user.domain.auth.exception
+
+import hs.kr.equus.user.global.error.exception.EquusException
+import hs.kr.equus.user.global.error.exception.ErrorCode
+
+object InvalidOkCertConnectException : EquusException(
+    ErrorCode.INVALID_OKCERT_CONNECTION
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/exception/InvalidOkCertConnectException.kt:7]: 🚫 **[ktlint]** <standard:trailing-comma-on-call-site> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing trailing comma before ")"

File: src/main/kotlin/hs/kr/equus/user/domain/auth/exception/InvalidPassException.kt [hunk 7]
```diff
@@ -0,0 +1,8 @@
+package hs.kr.equus.user.domain.auth.exception
+
+import hs.kr.equus.user.global.error.exception.EquusException
+import hs.kr.equus.user.global.error.exception.ErrorCode
+
+object InvalidPassException : EquusException(
+    ErrorCode.INTERNAL_SERVER_ERROR
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/exception/InvalidPassException.kt:7]: 🚫 **[ktlint]** <standard:trailing-comma-on-call-site> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing trailing comma before ")"

File: src/main/kotlin/hs/kr/equus/user/domain/auth/exception/InvalidUrlException.kt [hunk 8]
```diff
@@ -0,0 +1,8 @@
+package hs.kr.equus.user.domain.auth.exception
+
+import hs.kr.equus.user.global.error.exception.EquusException
+import hs.kr.equus.user.global.error.exception.ErrorCode
+
+object InvalidUrlException : EquusException(
+    ErrorCode.INVALID_URL
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/exception/InvalidUrlException.kt:7]: 🚫 **[ktlint]** <standard:trailing-comma-on-call-site> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing trailing comma before ")"

File: src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/PassInfoController.kt [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import javax.validation.Valid
+
+@RestController
+@RequestMapping("/user/verify")
+class PassInfoController(
+    private val passPopupService: PassPopupService,
+    private val queryPassInfoService: QueryPassInfoService
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/PassInfoController.kt:19]: 🚫 **[ktlint]** <standard:trailing-comma-on-declaration-site> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing trailing comma before ")"

File: src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/PassInfoController.kt [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+) {
+
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/PassInfoController.kt:21]: 🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line

File: src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/PassInfoController.kt [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+    @GetMapping("/info")
+    fun getPassInfo(@RequestParam("mdl_tkn") token: String): QueryPassInfoResponse = queryPassInfoService.execute(token)
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/PassInfoController.kt:23]: 🚫 **[ktlint]** <standard:parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should start on a newline

Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/PassInfoController.kt:23]: 🚫 **[ktlint]** <standard:function-signature> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Newline expected after opening parenthesis

Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/PassInfoController.kt:23]: 🚫 **[ktlint]** <standard:parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing newline before ")"

Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/PassInfoController.kt:23]: 🚫 **[ktlint]** <standard:function-signature> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Newline expected before closing parenthesis

File: src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/PassInfoController.kt [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+
+    @PostMapping("/popup")
+    fun getPass(@RequestBody request: @Valid PassPopupRequest): String = passPopupService.execute(request)
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/PassInfoController.kt:26]: 🚫 **[ktlint]** <standard:parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should start on a newline

Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/PassInfoController.kt:26]: 🚫 **[ktlint]** <standard:function-signature> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Newline expected after opening parenthesis

Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/PassInfoController.kt:26]: 🚫 **[ktlint]** <standard:parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing newline before ")"

Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/PassInfoController.kt:26]: 🚫 **[ktlint]** <standard:function-signature> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Newline expected before closing parenthesis

File: src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/dto/request/PassPopupRequest.kt [hunk 13]
```diff
@@ -0,0 +1,8 @@
+package hs.kr.equus.user.domain.auth.presentation.dto.request
+
+import javax.validation.constraints.NotBlank
+
+data class PassPopupRequest(
+    @NotBlank(message = "redirect_url은 Null 또는 공백 또는 띄어쓰기를 허용하지 않습니다.")
+    val redirectUrl: String
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/dto/request/PassPopupRequest.kt:7]: 🚫 **[ktlint]** <standard:trailing-comma-on-declaration-site> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing trailing comma before ")"

File: src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/dto/resopnse/QueryPassInfoResponse.kt [hunk 14]
```diff
@@ -0,0 +1,6 @@
+package hs.kr.equus.user.domain.auth.presentation.dto.resopnse
+
+data class QueryPassInfoResponse(
+    private val phoneNumber: String,
+    private val name: String
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/dto/resopnse/QueryPassInfoResponse.kt:5]: 🚫 **[ktlint]** <standard:trailing-comma-on-declaration-site> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing trailing comma before ")"

File: src/main/kotlin/hs/kr/equus/user/domain/auth/service/PassPopupService.kt [hunk 15]
```diff
@@ -0,0 +1,106 @@
...
+import hs.kr.equus.user.global.utils.pass.RedirectUrlChecker
+import kcb.module.v3.OkCert
+import org.json.JSONObject
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class PassPopupService(
+    private val redirectUrlChecker: RedirectUrlChecker
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/service/PassPopupService.kt:14]: 🚫 **[ktlint]** <standard:trailing-comma-on-declaration-site> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing trailing comma before ")"

File: src/main/kotlin/hs/kr/equus/user/domain/auth/service/PassPopupService.kt [hunk 16]
```diff
@@ -0,0 +1,106 @@
...
+) {
+
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/service/PassPopupService.kt:16]: 🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line

File: src/main/kotlin/hs/kr/equus/user/domain/auth/service/PassPopupService.kt [hunk 17]
```diff
@@ -0,0 +1,106 @@
...
+            val reqStr: String = reqJson.toString()
+
+            val okcert = OkCert()
+
+            val resultStr: String = okcert.callOkCert(TARGET, cpCd, svcName, license, reqStr)
+
+            val resJson = JSONObject(resultStr)
+
+            val RSLT_CD: String = resJson.getString("RSLT_CD")
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/service/PassPopupService.kt:58]: 🚫 **[ktlint]** <standard:property-naming> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Property name should start with a lowercase letter and use camel case

File: src/main/kotlin/hs/kr/equus/user/domain/auth/service/PassPopupService.kt [hunk 18]
```diff
@@ -0,0 +1,106 @@
...
+            val RSLT_MSG: String = resJson.getString("RSLT_MSG")
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/service/PassPopupService.kt:59]: 🚫 **[ktlint]** <standard:property-naming> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Property name should start with a lowercase letter and use camel case

File: src/main/kotlin/hs/kr/equus/user/domain/auth/service/PassPopupService.kt [hunk 19]
```diff
@@ -0,0 +1,106 @@
...
+            var MDL_TKN = ""
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/service/PassPopupService.kt:60]: 🚫 **[ktlint]** <standard:property-naming> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Property name should start with a lowercase letter and use camel case

File: src/main/kotlin/hs/kr/equus/user/domain/auth/service/PassPopupService.kt [hunk 20]
```diff
@@ -0,0 +1,106 @@
...
+            htmlBuilder.append("document.form1.action = \"$popupUrl\";")
+            htmlBuilder.append("document.form1.method = \"post\";")
+            htmlBuilder.append("document.form1.submit();")
+            htmlBuilder.append("}")
+            htmlBuilder.append("</script>")
+            htmlBuilder.append("</head>")
+            htmlBuilder.append("<body>")
+            htmlBuilder.append("<form name=\"form1\">")
+            htmlBuilder.append(
+                "<input type=\"hidden\" name=\"tc\" value=\"kcb.oknm.online.safehscert.popup.cmd.P931_CertChoiceCmd\"/>"
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/service/PassPopupService.kt:85]: 🚫 **[ktlint]** <standard:trailing-comma-on-call-site> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing trailing comma before ")"

File: src/main/kotlin/hs/kr/equus/user/domain/auth/service/QueryPassInfoService.kt [hunk 21]
```diff
@@ -0,0 +1,45 @@
...
+import hs.kr.equus.user.domain.auth.presentation.dto.resopnse.QueryPassInfoResponse
+import hs.kr.equus.user.global.utils.pass.PassUtil
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class QueryPassInfoService(
+    private val passInfoRepository: PassInfoRepository,
+    private val passUtil: PassUtil
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/service/QueryPassInfoService.kt:15]: 🚫 **[ktlint]** <standard:trailing-comma-on-declaration-site> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing trailing comma before ")"

File: src/main/kotlin/hs/kr/equus/user/domain/auth/service/QueryPassInfoService.kt [hunk 22]
```diff
@@ -0,0 +1,45 @@
...
+) {
+
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/service/QueryPassInfoService.kt:17]: 🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line

File: src/main/kotlin/hs/kr/equus/user/domain/auth/service/QueryPassInfoService.kt [hunk 23]
```diff
@@ -0,0 +1,45 @@
...
+    companion object {
+
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/service/QueryPassInfoService.kt:19]: 🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line

File: src/main/kotlin/hs/kr/equus/user/domain/auth/service/QueryPassInfoService.kt [hunk 24]
```diff
@@ -0,0 +1,45 @@
...
+        private val RESULT_NAME = "RSLT_NAME"
+
+        private val TEL_NO = "TEL_NO"
+
+        private val RESULT_CODE_OK = "B000"
+    }
+
+    @Value("\${pass.exp}")
+    private var EXP: Long = 0L
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/domain/auth/service/QueryPassInfoService.kt:30]: 🚫 **[ktlint]** <standard:property-naming> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Property name should start with a lowercase letter and use camel case

**Review by HyunSu1768 — COMMENTED** [review]
Remaining comments which cannot be posted as a review comment to avoid GitHub Rate Limit

<details>
<summary>ktlint</summary>

[src/main/kotlin/hs/kr/equus/user/global/error/GlobalExceptionHandler.kt|29 col 35|](https://github.com/EntryDSM/Equus-User/blob/95241509ca2b4c8a29ad8ce11a7d75787a6b3778/src/main/kotlin/hs/kr/equus/user/global/error/GlobalExceptionHandler.kt#L29) Missing trailing comma before ")"
[src/main/kotlin/hs/kr/equus/user/global/error/exception/ErrorCode.kt|17 col 83|](https://github.com/EntryDSM/Equus-User/blob/95241509ca2b4c8a29ad8ce11a7d75787a6b3778/src/main/kotlin/hs/kr/equus/user/global/error/exception/ErrorCode.kt#L17) Missing trailing comma before "}"
[src/main/kotlin/hs/kr/equus/user/global/exception/InternalServerErrorException.kt|7 col 36|](https://github.com/EntryDSM/Equus-User/blob/95241509ca2b4c8a29ad8ce11a7d75787a6b3778/src/main/kotlin/hs/kr/equus/user/global/exception/InternalServerErrorException.kt#L7) Missing trailing comma before ")"
[src/main/kotlin/hs/kr/equus/user/global/security/SecurityConfig.kt|13 col 1|](https://github.com/EntryDSM/Equus-User/blob/95241509ca2b4c8a29ad8ce11a7d75787a6b3778/src/main/kotlin/hs/kr/equus/user/global/security/SecurityConfig.kt#L13) Class body should not start with blank line
[src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt|12 col 1|](https://github.com/EntryDSM/Equus-User/blob/95241509ca2b4c8a29ad8ce11a7d75787a6b3778/src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt#L12) Class body should not start with blank line
[src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt|14 col 1|](https://github.com/EntryDSM/Equus-User/blob/95241509ca2b4c8a29ad8ce11a7d75787a6b3778/src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt#L14) Class body should not start with blank line
[src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt|25 col 26|](https://github.com/EntryDSM/Equus-User/blob/95241509ca2b4c8a29ad8ce11a7d75787a6b3778/src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt#L25) Property name should start with a lowercase letter and use camel case
[src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt|28 col 26|](https://github.com/EntryDSM/Equus-User/blob/95241509ca2b4c8a29ad8ce11a7d75787a6b3778/src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt#L28) Property name should start with a lowercase letter and use camel case
[src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt|35 col 21|](https://github.com/EntryDSM/Equus-User/blob/95241509ca2b4c8a29ad8ce11a7d75787a6b3778/src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt#L35) A multiline expression should start on a new line
[src/main/kotlin/hs/kr/equus/user/global/utils/pass/RedirectUrlChecker.kt|9 col 1|](https://github.com/EntryDSM/Equus-User/blob/95241509ca2b4c8a29ad8ce11a7d75787a6b3778/src/main/kotlin/hs/kr/equus/user/global/utils/pass/RedirectUrlChecker.kt#L9) Class body should not start with blank line
[src/main/kotlin/hs/kr/equus/user/global/utils/pass/RedirectUrlChecker.kt|11 col 26|](https://github.com/EntryDSM/Equus-User/blob/95241509ca2b4c8a29ad8ce11a7d75787a6b3778/src/main/kotlin/hs/kr/equus/user/global/utils/pass/RedirectUrlChecker.kt#L11) Property name should start with a lowercase letter and use camel case
</details>

File: src/main/kotlin/hs/kr/equus/user/global/error/GlobalExceptionHandler.kt [hunk 25]
```diff
@@ -25,7 +25,8 @@ class GlobalExceptionHandler {
             ErrorResponse(
                 400,
                 e.bindingResult.allErrors[0].defaultMessage
-            ), HttpStatus.BAD_REQUEST
+            ),
+            HttpStatus.BAD_REQUEST
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/global/error/GlobalExceptionHandler.kt:29]: 🚫 **[ktlint]** <standard:trailing-comma-on-call-site> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing trailing comma before ")"

File: src/main/kotlin/hs/kr/equus/user/global/error/exception/ErrorCode.kt [hunk 26]
```diff
@@ -9,7 +9,10 @@ enum class ErrorCode(
...
     UNAUTHENTICATED(401, "COMMON401-2", "UnAuthenticated"),
+    INVALID_URL(401, "URL401-0", "Invalid Url"),
+    INVALID_PASS(401, "PASS_INFO-401-0", "Invalid Pass"),
 
     // Internal Server Error
-    INTERNAL_SERVER_ERROR(500, "COMMON500-0", "Internal Server Error")
-}
\ No newline at end of file
+    INTERNAL_SERVER_ERROR(500, "COMMON500-0", "Internal Server Error"),
+    INVALID_OKCERT_CONNECTION(500, "PASS_INFO-500-0", "Invalid OkCert Connection")
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/global/error/exception/ErrorCode.kt:17]: 🚫 **[ktlint]** <standard:trailing-comma-on-declaration-site> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing trailing comma before "}"

File: src/main/kotlin/hs/kr/equus/user/global/exception/InternalServerErrorException.kt [hunk 27]
```diff
@@ -0,0 +1,8 @@
+package hs.kr.equus.user.global.exception
+
+import hs.kr.equus.user.global.error.exception.EquusException
+import hs.kr.equus.user.global.error.exception.ErrorCode
+
+object InternalServerErrorException : EquusException(
+    ErrorCode.INTERNAL_SERVER_ERROR
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/global/exception/InternalServerErrorException.kt:7]: 🚫 **[ktlint]** <standard:trailing-comma-on-call-site> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing trailing comma before ")"

File: src/main/kotlin/hs/kr/equus/user/global/security/SecurityConfig.kt [hunk 28]
```diff
@@ -0,0 +1,37 @@
...
+import org.springframework.context.annotation.Configuration
+import org.springframework.http.HttpMethod
+import org.springframework.security.config.annotation.web.builders.HttpSecurity
+import org.springframework.security.config.http.SessionCreationPolicy
+import org.springframework.security.web.SecurityFilterChain
+import org.springframework.web.cors.CorsUtils
+
+@Configuration
+class SecurityConfig {
+
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/global/security/SecurityConfig.kt:13]: 🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line

File: src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt [hunk 29]
```diff
@@ -0,0 +1,43 @@
...
+import hs.kr.equus.user.domain.auth.exception.InvalidOkCertConnectException
+import kcb.module.v3.OkCert
+import kcb.module.v3.exception.OkCertException
+import kcb.org.json.JSONObject
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+
+@Component
+class PassUtil {
+
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt:12]: 🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line

File: src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt [hunk 30]
```diff
@@ -0,0 +1,43 @@
...
+    companion object {
+
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt:14]: 🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line

File: src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt [hunk 31]
```diff
@@ -0,0 +1,43 @@
...
+        private val TARGET = "PROD"
+
+        private val SVC_NAME = "IDS_HS_POPUP_RESULT"
+
+        private val MODEL_TOKEN = "MDL_TKN"
+    }
+
+    @Value("{pass.cp-cd}")
+    private lateinit var CP_CD: String
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt:25]: 🚫 **[ktlint]** <standard:property-naming> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Property name should start with a lowercase letter and use camel case

File: src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt [hunk 32]
```diff
@@ -0,0 +1,43 @@
...
+
+    @Value("{pass.license}")
+    private lateinit var LICENSE: String
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt:28]: 🚫 **[ktlint]** <standard:property-naming> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Property name should start with a lowercase letter and use camel case

File: src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt [hunk 33]
```diff
@@ -0,0 +1,43 @@
...
+
+    fun getResponseJson(token: String?): JSONObject? {
+        val reqJson = JSONObject()
+        reqJson.put(MODEL_TOKEN, token)
+        val reqStr = reqJson.toString()
+        var resultStr: String? = null
+        resultStr = try {
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt:35]: 🚫 **[ktlint]** <standard:multiline-expression-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>A multiline expression should start on a new line

File: src/main/kotlin/hs/kr/equus/user/global/utils/pass/RedirectUrlChecker.kt [hunk 34]
```diff
@@ -0,0 +1,18 @@
+package hs.kr.equus.user.global.utils.pass
+
+import hs.kr.equus.user.domain.auth.exception.InvalidUrlException
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+
+@Component
+class RedirectUrlChecker {
+
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/global/utils/pass/RedirectUrlChecker.kt:9]: 🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line

File: src/main/kotlin/hs/kr/equus/user/global/utils/pass/RedirectUrlChecker.kt [hunk 35]
```diff
@@ -0,0 +1,18 @@
...
+    @Value("\${pass.base-url}")
+    private lateinit var BASE_URL: String
```
Comment (HyunSu1768) [inline src/main/kotlin/hs/kr/equus/user/global/utils/pass/RedirectUrlChecker.kt:11]: 🚫 **[ktlint]** <standard:property-naming> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Property name should start with a lowercase letter and use camel case

File: src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/PassInfoController.kt (hunk 12 above)
Comment (lilseongwon) [reply src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/PassInfoController.kt:25]: Postmapping이니까 getPass란 이름보다 popupPass라는 메서드를 사용하는 게 좋을듯

File: src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt (hunk 33 above)
Comment (lilseongwon) [reply src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt:35]: 이거는 Nullable로 하면 안되지 않나?

Comment (HyunSu1768) [reply src/main/kotlin/hs/kr/equus/user/global/utils/pass/PassUtil.kt]: 수정 했습니다 😊

**Review by jyk1029 — APPROVED** [review]

**Review by lilseongwon — APPROVED** [review]

**Pull request merged**